config/routes.rb

```diff
@@ -7,6 +7,7 @@
       resources :device_registrations, only: %i[create]
       mount_devise_token_auth_for 'User', at: 'auth'
       post '/auth/login', to: 'authentication#login'
+      resources :feed, only: %i[index]
```

Comment:
Once again, use `resource feed, only: %[show]` instead o this.

Reply:
si pongo show me pide id en la ruta lo cual esta mal

Reply:
No it won't ^^

<img width="1101" alt="image" src="https://user-images.githubusercontent.com/11707739/96792579-0a7d2b80-13d1-11eb-8a93-9c9899a84484.png">

Reply:
![Captura de Pantalla 2020-10-22 a la(s) 10 55 10](https://user-images.githubusercontent.com/67917450/96881955-27f3d900-1455-11eb-907a-de2ef9bd3d04.png)